tests/Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot

```diff
@@ -66,7 +66,20 @@ JupyterLab Code Cell Error Output Should Not Be Visible
 
 Get JupyterLab Code Cell Error Text
   ${error_txt} =  Get Text  //div[contains(@class,"jp-OutputArea-output") and @data-mime-type="application/vnd.jupyter.stderr"]
-  [Return]
+  [Return]  ${error_txt}
+
+Run Git Repo And Return Last Cell Error Text
+  [Documentation]    It actually clones the git repo, runs it and then returns the error
+  [Arguments]    ${REPO_URL}  ${NOTEBOOK_TO_RUN}
+  Run Keyword And Ignore Error    Clone Git Repository And Run    ${LINK_OF_GITHUB}    ${PATH_TO_FILE}
+  ${output} =    Get JupyterLab Code Cell Error Text
+  [Return]    ${output}
+
+Error Test Should Be Expected Text
```

Comment:
hm maybe the title is a bit confusing. maybe you could use something like `Cell Error Message Should Be`